### PR TITLE
[W-11288298] - Accesibility bug fix

### PR DESCRIPTION
**Pull request by maurimart**
[Technical Design Document W-11288298](https://salesforce.quip.com/0HYbAcObiRDJl)

# Critical Changes

Navigate table using the arrow keys to move through the cells.
Remove link like hovering for headers 
Change first element focus for first element, focus trap on modals.

# Changes

### CODE REVIEW

File: force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.html [hunk 1]
```diff
@@ -123,25 +123,26 @@
             <tbody>
                 <template for:each={data} for:item="item">
                     <tr key={item.recurringDonation.Id} aria-selected="false" class="slds-hint-parent" scope="row">
-                        <td key={item.recurringDonation.npe03__Amount__c} data-label={labels.commonAmount}>
+                        <td key={item.recurringDonation.npe03__Amount__c} data-label={labels.commonAmount} tabindex="-1" class="move">
```
Comment (lparrott) [inline force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.html:126]: What is the "move" class for? I can't find reference to it in JS or CSS.

**Review by lparrott — APPROVED** [review]
I'm all for accessibility fixes, but we usually try to rely on Lightning Components to handle this for us. I haven't tested the changes here, but they seem ok.

**Review by gerardorodriguezsf — APPROVED** [review]
LGTM

File: force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.js [hunk 2]
```diff
@@ -327,7 +351,9 @@ export default class RecurringDonationTable extends LightningElement {
     }
 
     getRecurringDonationFields() {
+        console.log('data retrieve');
```
Comment (JoseRossiSF) [inline force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.js:354]: Please remove the console.log

File: force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.js [hunk 3]
```diff
@@ -327,7 +351,9 @@ export default class RecurringDonationTable extends LightningElement {
...
         retrieveTableView().then((data) => {
+            console.log(data);
```
Comment (JoseRossiSF) [inline force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.js:356]: Please remove the console.log

**Review by JoseRossiSF — APPROVED** [review]
Please remove the console.logs. Besides that, it looks good to me.

**Pull request merged**